xActiveDirectory.psd1

```diff
@@ -1,6 +1,6 @@
 @{
 # Version number of this module.
-ModuleVersion = '2.9.0.0'
```

Comment:
Please do not change the module or class version in resources per the [contribution guidelines](https://github.com/PowerShell/DscResources/blob/master/CONTRIBUTING.md#creating-a-new-dsc-resource-in-an-existing-module)